rest/v10/webhook.ts

```diff
@@ -144,6 +144,13 @@ export type RESTPostAPIWebhookWithTokenJSONBody = AddUndefinedToPossiblyUndefine
 	 * Message flags combined as a bitfield
 	 */
 	flags?: MessageFlags;
+	/**
+	 * Name of thread to create (only available if webhook is in a forum channel)
+	 *
+	 * If the webhook is in a thread channel, you must provide either `thread_id` in the query string params, or `thread_name` in the JSON/form params.
+	 * If `thread_id` is provided, the message will send in that thread. If `thread_name` is provided, a thread with that name will be created in the forum channel.
```

Comment:
This should use `{@link Type.Property}`, since `thread_name` and `thread_id` are from different interfaces.

Because the note is from outside the table, if we want to mention it, it should be largely rewritten to be contextful.

Reply:
![image](https://user-images.githubusercontent.com/79790819/171573822-8b668802-4ac4-430c-9492-71657e211194.png)
![image](https://user-images.githubusercontent.com/79790819/171573831-d1faf1a9-2574-42ce-9951-21bd52df328f.png)

Would this be ok?

Reply:
Yes, it is ok.